<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LARCmaCS</class>
 <widget class="QWidget" name="LARCmaCS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LARCmaCS</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="50,0,0,1">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="10,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGraphicsView" name="fieldView"/>
       </item>
       <item>
        <widget class="QSlider" name="sceneslider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
         <property name="sliderPosition">
          <number>8</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="but_reference">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../bin/help.png</normaloff>../bin/help.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox_SimEnable">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Enable Simulator</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Simulator IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSimIP">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Blue port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSimPort">
           <property name="text">
            <string>10301</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Yellow port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSimPortYellow">
           <property name="text">
            <string>10302</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2_1">
           <property name="text">
            <string>Network interface</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditNetInterface">
           <property name="text">
            <string>eno1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Robot</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Connection</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Charge</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="num1">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="connection1">
             <property name="text">
              <string>No</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="battery1">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="num2">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="connection2">
             <property name="text">
              <string>No</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="battery2">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="num3">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="connection3">
             <property name="text">
              <string>No</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="battery3">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="num4">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="connection4">
             <property name="text">
              <string>No</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="battery4">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="num5">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="connection5">
             <property name="text">
              <string>No</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="battery5">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="num6">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="connection6">
             <property name="text">
              <string>No</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="battery6">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="matlabConsoleLabel">
       <property name="text">
        <string>Matlab Console |  Update every</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="matlabOutputFrequencyLineEdit">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>100</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>frame(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="matlabConsole">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_SetMLdir">
         <property name="text">
          <string>Set MatLab Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Pause">
         <property name="text">
          <string>Matlab_Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Pause">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="text">
          <string>---</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditRobotIp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>192.168.43.10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_RemoteControl">
         <property name="text">
          <string>Remote control</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="10,10,1">
     <item>
      <widget class="QLabel" name="StatusLabel">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_SSL_FPS">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelscenesize">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sceneslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelscenesize</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
